tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb

```diff
@@ -6,9 +6,21 @@ class UploadSymbolsToBugsnagAction < Action
         File.join(File.dirname(__FILE__), '..', '..', '..', '..', '..', 'bugsnag-dsym-upload'))
 
       def self.run(params)
+        options = {}
+        options = options_from_info_plist(params[:config_file]) if params[:config_file]
+
+        if (params[:config_file] == default_info_plist_path)
+          # Load custom API key if config file has not been overridden
+          options[:apiKey] = params[:api_key] unless params[:api_key].nil?
+        else
+          # Print which file is populating version and API key information since the value has been
+          # overridden
+          UI.message("Loading API key from #{params[:config_file]}")
```

Comment:
What happens if the `config_file` option is present but the `BugsnagApiKey` is not inside it?

Reply:
Tested this - if you have a `config_file` parameter with no API key, the `api_key` parameter will be ignored.  We should change this to use the `api_key` parameter if not api key can be extracted from `info.plist`.

Reply:
Added a test for this, will now use the parameter if there is nothing in the config file.